app/components/works/form_component.rb

```diff
@@ -29,5 +31,17 @@ def work
     def work_version
       work_form.model.fetch(:work_version)
     end
+
+    def doi_field
+      return "https://doi.org/#{Doi.for(druid)}." if doi
```

Comment:
Shouldn't we just return the DOI if it is present?  Some DOIs do not match the druid patter as they were issued outside of H2.

Reply:
Oops hadn't been thinking about other DOIs. Fixed.